README.md

```diff
@@ -20,13 +22,14 @@ optional arguments:
   -s, --southbound      Use this if you're going southbound
   -w WALK_TIME, --walk-time WALK_TIME
                         The walk time (in minutes) from where you are to the
-                        departure station
+                        departure station.
+			If you omit this argument, your walk-time will be estimated based on your current location.
 ```
 
 ## Example
-You want to leave from Tara Street, go southbound, and the walk-time to the station is 10 mins
+You want to leave from Tara Street, go southbound.
 ```
-$ ./dart.py "Tara Street" --southbound --walk-time 10
+$ ./dart.py "Tara Street" --southbound
```

Comment:
Maybe add an example with the walk time and an example with the google distance stuff?